dolphinscheduler-dao/src/main/resources/sql/upgrade/3.3.0_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -125,3 +125,5 @@ d//
 delimiter ;
 CALL drop_column_t_ds_alert_plugin_instance;
 DROP PROCEDURE drop_column_t_ds_alert_plugin_instance;
+
+ALTER TABLE t_ds_schedules ADD  `modify_user_id` int(11) DEFAULT NULL COMMENT 'modify user id';
```

Comment:
Please leave a blank line at the end of each file.

Reply:
> Please leave a blank line at the end of each file.

done